src/components/notifications/HostedSMS.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <div class="mb-3">
+        <label for="hostedsms-username" class="form-label">{{ $t('hostedsmsAPIUser') }}</label>
+        <input id="hostedsms-username" v-model="$parent.notification.hostedsmsUsername" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="hostedsms-key" class="form-label">{{ $t('hostedsmsAPIPassword') }}</label>
+        <HiddenInput id="hostedsms-key" v-model="$parent.notification.hostedsmsPassword" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="hostedsms-phone-number" class="form-label">{{ $t("hostedsmsPhoneNumber") }}</label>
+        <input id="hostedsms-phone-number" v-model="$parent.notification.hostedsmsPhoneNumber" type="text" class="form-control" required>
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel
```suggestion
        <input id="hostedsms-phone-number" v-model="$parent.notification.hostedsmsPhoneNumber" type="tel" class="form-control" required>
```